qiskit/providers/aer/backends/wrappers/qasm_bindings.cc

```diff
@@ -0,0 +1,13 @@
+#include <pybind11/pybind11.h>
+
+#include "simulators/qasm/qasm_controller.hpp"
+#include "simulators/controller_execute.hpp"
+#include <iostream>
+#include "framework/json.hpp"
+
+PYBIND11_MODULE(qasm_controller_wrapper, m) {
```

Comment:
When switching to PyBind should these files remain in the qiskit folder or be moved somewhere into `src` folder? ie `src/pybind`?

Reply:
We could go either way - would just need to adjust the CMakeLists.txt 
- benefit of the current structure is that the source tree reflects the eventual install tree structure 
- benefit of moving would be keeping all c++ source code together 
There are pluses and minuses both ways but whichever you feel is better I can make the changes

Reply:
Going along with this - we could also combine the three qasm, unitary, statevector wrappers into a single file / module - more benefits in terms of maintenance here, same drawbacks of no longer reflecting the install tree structure